server/datastore/mysql/software_installers.go

```diff
@@ -184,22 +184,30 @@ WHERE
 func (ds *Datastore) GetSoftwareInstallerMetadataByTeamAndTitleID(ctx context.Context, teamID *uint, titleID uint) (*fleet.SoftwareInstaller, error) {
 	query := `
 SELECT
-	si.id,
-	si.team_id,
-	si.title_id,
-	si.storage_id,
-	si.filename,
-	si.version,
-	si.install_script_content_id,
-	si.pre_install_query,
-	si.post_install_script_content_id,
-	si.uploaded_at,
-	COALESCE(st.name, '') AS software_title
+  si.id,
+  si.team_id,
+  si.title_id,
+  si.storage_id,
+  si.filename,
+  si.version,
+  si.install_script_content_id,
+  si.pre_install_query,
+  si.post_install_script_content_id,
+  si.uploaded_at,
+  inst.contents AS install_script,
+  COALESCE(pisnt.contents, '') AS post_install_script,
+  COALESCE(st.name, '') AS software_title
 FROM 
-	software_installers si
-	LEFT OUTER JOIN software_titles st ON st.id = si.title_id
+  software_installers si
+  LEFT OUTER JOIN software_titles st ON st.id = si.title_id
+  LEFT OUTER JOIN
+    script_contents inst
+    ON inst.id = si.install_script_content_id
+  LEFT OUTER JOIN
+    script_contents pisnt
+    ON pisnt.id = si.post_install_script_content_id
```

Comment:
Do we also need to update the other datastore method (the one that that gets metadata by installer id)?

Reply:
good question, not currently. I think we can even potentially get rid of it when we catch a breather.